Resources/Textures/Objects/Fun/toys.rsi/meta.json

```diff
@@ -365,6 +365,9 @@
     },
     {
       "name": "rubber_chicken"
+    },
+    {
+      "name": "mortplush"
```

Comment:
add it to `Resources/Textures/DeltaV/Objects/Fun/toys.rsi` and add/update the copyright to say its created by you or whatever